eisbuk-admin/src/store/selectors/calendar.ts

```diff
@@ -0,0 +1,49 @@
+import { LocalStore } from "@/types/store";
+
+interface calendarSlot {
+  [dayInISO: string]: string;
+}
+
+export const getCalendarData = (state: LocalStore): calendarSlot => {
+  const {
+    firestore: {
+      data: { slotsByDay: allSlotsInStore, attendance = {}, bookedSlots = {} },
+    },
+  } = state;
+
+  const slotsByDay = allSlotsInStore ?? {};
```

Comment:
`const slotsByDay = allSlotsInStore ?? {}` is the same as declaring
```
const { firestore: {
    data: { slotsByDay = {} }
  }
} = state
```
in both cases, the slotsByDay will fall back to empty `{}` if `undefined | null`

Reply:
Scratch that...it will only work if undefined, not if `null`